style.css

```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  letter-spacing: 2px;
+}
+html {
+  font-size: 15px;
+}
+body {
+  background-color: #000000;
+}
+header {
+  width: 100%;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+}
+h1 {
+  color: #c0c0c0;
+}
+body {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+.card-section {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  max-width: fit-content;
+  justify-content: center;
+}
+.card {
+  border: 1px solid black;
+  border-radius: 20px;
+  margin: 5px;
+  padding: 0 20px;
+  width: 40%;
+  flex-direction: column;
+  display: flex;
+  background-color: #ff8c00;
+}
+@media screen and (max-width: 800px) {
+  .card-section {
+    flex-direction: column;
+  }
```

Comment:
needs auto margins on sides to be centred after you remove display flex from body